x/ibc/02-client/handler.go

```diff
@@ -38,8 +38,9 @@ func HandleMsgCreateClient(ctx sdk.Context, k keeper.Keeper, msg exported.MsgCre
 		consensusHeight = msg.GetConsensusState().GetHeight()
 	case exported.Localhost:
 		// msg client id is always "localhost"
+		// Localhost epoch is always 0 for now
```

Comment:
@cwgoes I think this needs a note on the ICS09 spec

Reply:
hmm, localhost clients shouldn't even have an epoch

Reply:
nor should solo machine clients